doc/rst/tools/chplcheck/chplcheck.rst

```diff
@@ -0,0 +1,288 @@
+.. _readme-chplcheck:
+
+chplcheck
+=========
+
+``chplcheck`` is a linter for the Chapel programming language implemented in
+Python using the :ref:`Python bindings for the compiler frontend <readme-chapel-py>`.
+It is intended to catch stylistic mistakes and bad practices in Chapel programs.
+It is also intended to be customizable and extensible, using a system of named
+'rules' that lead to warnings.
+
+``chplcheck`` supports the Language Server Protocol, allowing it to be used as
+part of your favorite editor. The following image demonstrates its' use in
+Neovim:
+
+.. image:: neovim.png
+  :alt: Screenshot of code using ``chplcheck``
+
+Getting Started
+---------------
+
+The easiest way to make ``chplcheck`` available on your command line is by using the
+``chplcheck`` Makefile target. This will build the Dyno compiler frontend and the
+Python bindings for Dyno if needed, and place ``chplcheck`` into ``$CHPL_HOME/bin``.
+In from-source builds, this directory should already be in your ``PATH``;
+this means you should be invoke ``chplcheck`` from the command-line without any
+additional work.
```

Comment:
I can see now that you took inspiration from this section in CLS -- and I think this one needs the "build from source" qualifier too.